app/importers/templates/media-size-importer.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Template for Activity Upgrade - RTMediaMediaSizeImporter::init().
+ *
+ * @package rtMedia
+ */
+
+?>
+
+<div class="wrap">
+	<h2><?php esc_html_e( 'rtMedia: Import Media Size', 'buddypress-media' ); ?></h2>
+	<?php
+	wp_nonce_field( 'rtmedia_media_size_import_nonce', 'rtmedia_media_size_import_nonce' );
+
+	printf(
+	// translators: %2$s: Estimated.
```

Comment:
This comment not needed because there is no placeholder used in translatable string. So remove it.

Reply:
Done